atom/browser/api/atom_api_window.cc

```diff
@@ -400,6 +405,11 @@ void Window::SetProgressBar(double progress) {
   window_->SetProgressBar(progress);
 }
 
+void Window::SetOverlayIcon(const gfx::Image& overlay,
+  const std::string& description) {
```

Comment:
The parameters of functions should be aligned, see:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Function_Declarations_and_Definitions